app/views/inventory_foods/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="food-view">
+<div class="d-flex justify-content-end me-5 p-2">
+<%= link_to 'Add Food', new_inventory_inventory_food_path( @inventory.id), class: "btn btn-dark p-3" %>
+</div>
+<hr>
+<table class=" table table-striped table-bordered table-hover text-center" style="width: 80%; margin-top: 3re; margin-inline:auto">
+  <thead class="thead-dark">
+    <tr>
+      <th scope="col">#</th>
+      <th scope="col">Food</th>
+      <th scope="col">Quantity</th>
+      <th scope="col">Action</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% InventoryFood.all.each do |infood| %>
+        <tr>
+            <th scope="row"><%= infood.id %></th>
+            <td><%= infood.food.name %></td>
+            <td><%= infood.quantity %></td>
+            <td>
+            
+            <%= button_to "Delete", inventory_inventory_food_path(@inventory, infood), data: {turbo: false}, method: :delete, class: "btn btn-danger" %>
+            
+            </td>
```

Comment:
- [ ] For your inventory, Though the adding of inventory works there is an exception and we need to address that. Besides that, your design needs some adjustments to look like the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/inventory_index.png?raw=true). Please fix the exception and update the design.
- [ ] Also, Clicking on an inventory item, should lead to the form that allows users to add new food.
- [ ] Aside from that, don't forget to add a working `Remove inventory` button.
### Your design as of now 

https://user-images.githubusercontent.com/69077061/219879091-290c3b96-2e89-4408-a5fe-dbbf1dc4a1b3.mp4